### PR TITLE
feat: mp-units support added

### PR DESCRIPTION
**Pull request by mpusz**
Specify library name and version:  **mp-units/0.6.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by uilianries — APPROVED** [review]

**Review by czoido — DISMISSED** [review]

File: recipes/mp-units/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    
+    # TODO remove when https://github.com/conan-io/conan/issues/7680 is solved (or VS2019 is updated to at least 16.7)
+    def _skip_check(self):
+        return self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) <= 16
```
Comment (uilianries) [inline recipes/mp-units/all/test_package/conanfile.py:12]:
```suggestion
        return self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) <= "16"
```

**Review by danimtb — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]
👍 

We need to update MSBuild so this recipe is actually tested. I've added the issue here (https://github.com/conan-io/conan-center-index/issues/2920), we should come back to this recipe and test it as soon as it is upgraded.

**Comment by danimtb** [comment]
yay! 🚀 

```
$ conan search mp-units* -r conan-center
Existing package recipes:

mp-units/0.6.0
```

**Comment by mpusz** [comment]
BIG thanks to all the Conan team :-D You are the BEST!!!

**Pull request merged**